modules/accredited_representative_portal/app/models/accredited_representative_portal/saved_claim/benefits_intake.rb

```diff
@@ -59,6 +59,18 @@ module BusinessLines
       end
 
       delegate :to_pdf, to: :form_attachment
+
+      def parsed_form
+        JSON.parse(form)
+      end
+
+      def claimant_info
+        parsed_form.key?('dependent') ? parsed_form['dependent'] : parsed_form['veteran']
+      end
+
+      def latest_lighthouse_submission
+        lighthouse_submissions.order(created_at: :desc).first
+      end
```

Comment:
We inherit the write path for the asynchronous submission to Benefits Intake API and the creation of a corresponding bookkeeping record. Its definition is [here](https://github.com/department-of-veterans-affairs/vets-api/blob/e55baae33f94eae0fdf6cee22f9913e2b0856525/app/sidekiq/lighthouse/submit_benefits_intake_claim.rb#L130). So we are not using the table `lighthouse_submissions`. That was a [very recent data model fork from a different product team](https://github.com/department-of-veterans-affairs/vets-api/pull/21723).

Reply:
This echoes the ticket's enumeration of the underlying tables / associations:
```
saved_claims
  ar_saved_claim_claimant_representatives
  persistent_attachments
  form_submissions
    form_submission_attempts
```